### DISCUSSION TITLE
Is there a reason to use `setup_ci` on Gitlab Runners on Mac?

### DISCUSSION THREAD

**Discussion by allidoisace** [fastlane/fastlane#20092] [Q&A]
I am struggling to understand why we need to use `setup_ci`. All it does it force a prompt to sign in to the temporary keychain and that prevents the CI from running.

**Comment by Carissa34** [comment]
Backward compatibility is guaranteed between minor version updates. However, sometimes minor version updates of GitLab can introduce new features that require GitLab Runner to be on the same minor version

**Comment by CaseyCw1** [comment]
Thanks for the information, I will try to figure it out for more. Keep sharing such informative post keep suggesting such post.

[www.bookiemarket.com
](https://www.bookiemarket.net/)
